py/client/tests/test_updateby.py

```diff
@@ -35,12 +35,15 @@ def test_ema(self):
                                    on_nan=BadDataBehavior.RESET,
                                    big_value_context=MathContext.UNLIMITED)
 
-        ops = [ema_tick_decay(decay_ticks=100, cols=["ema_a = a"]),
-               ema_tick_decay(decay_ticks=100, cols=["ema_a = a"], op_control=op_ctrl),
-               ema_time_decay(ts_col="Timestamp", decay_time=10, cols=["ema_a = a"]),
-               ema_time_decay(ts_col="Timestamp", decay_time=1000000, cols=["ema_c = c"],
-                              op_control=op_ctrl),
-               ]
+        ops = [
+            ema_tick_decay(decay_ticks=100, cols="ema_a = a"),
+            ema_tick_decay(decay_ticks=100, cols="ema_a = a", op_control=op_ctrl),
+            ema_time_decay(ts_col="Timestamp", decay_time=10, cols="ema_a = a"),
+            ema_time_decay(ts_col="Timestamp", decay_time=1000000, cols=["ema_c = c"], op_control=op_ctrl),
+            ema_time_decay(ts_col="Timestamp", decay_time="00:00:00.001", cols="ema_c = c", op_control=op_ctrl),
+            ema_time_decay(ts_col="Timestamp", decay_time="PT00:00:00.001", cols="ema_c = c", op_control=op_ctrl),
+            ema_time_decay(ts_col="Timestamp", decay_time="PT1M", cols="ema_c = c", op_control=op_ctrl),
```

Comment:
output col names are being reused

Reply:
Yes, but the python tests iterate over these operations (vs. combining into a single `update_by` call) so the names don't collide in the output.

Perhaps more combined operator testing is needed, will add an additional test composed of different types of operators.